assembl/static2/js/app/components/cookies/legalContentsLinksList.spec.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+/* eslint-disable import/no-extraneous-dependencies */
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+/* eslint-enable */
+
+import LegalContentsLinksList from './legalContentsLinksList';
+
+configure({ adapter: new Adapter() });
```

Comment:
We should really have this configure adapter stuff in `tests/setupTests.js` to not copy paste it in each file. If you move it there, modify all existing files that use it.